sources/bing_webmaster/__init__.py

```diff
@@ -6,26 +6,27 @@
 """
 
 import time
-from typing import Iterable, Iterator, List, Sequence
+from typing import Dict, Iterator, List, Sequence
 
 import dlt
 from dlt.common import logger
-from dlt.common.typing import DictStrAny, DictStrStr
+from dlt.common.typing import DictStrAny
 from dlt.sources import DltResource
 
 from .helpers import get_stats_with_retry, parse_response
 
 
 @dlt.source(name="bing_webmaster")
 def source(
-    site_urls: List[str] = None, site_url_pages: Iterable[DictStrStr] = None
+    site_urls: List[str] = dlt.config.value,
```

Comment:
these should be "optional", no?

Reply:
Yeah, you are right. But we have to update the docstring of this source then, it makes it seem as though they are optional.